src/components/nav-menu-item/nav-menu-item.tsx

```diff
@@ -431,48 +510,52 @@ export class CalciteNavMenuItem {
         </div>
         {this.iconEnd && !this.editingActive && this.renderIconElEnd()}
         {this.editable && !this.editingActive && this.renderEditIcon()}
-        {!this.href && this.hasSubMenu ? this.renderDropdownIcon() : null}
-        {this.breadcrumb ? this.renderBreadcrumbIcon() : null}
+        {!this.href && this.hasSubMenu ? this.renderDropdownIcon(dir) : null}
+        {this.breadcrumb ? this.renderBreadcrumbIcon(dir) : null}
       </Fragment>
     );
   }
 
   render() {
+    const dir = getElementDir(this.el);
     return (
-      <Host>
+      <Host onFocus={this.focusHandler}>
         <li
-          class={`container ${
-            this.topLevelLayout === "vertical" ? "nav-item-vertical-parent" : ""
-          }`}
+          class={{
+            container: true,
+            "nav-item-vertical-parent": this.topLevelLayout === "vertical"
+          }}
           role="none"
         >
           <div class="item-content">
             <a
-              aria-current="todo"
+              aria-current="page"
```

Comment:
"Current" is currently being read back on all nav menu items, which is great to have for the selected menu item, but not properly identified for nav menu items which are not selected.

Can a conditional be added where the  `aria-current="page"` attribute is read by the currently active page, and other non-selected menu items are set to `aria-current="false"`?

(Similar to this [MDN example](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#example).)

Reply:
added an conditional where the `aria-current ="page"` when the nav-menu-item is focused.